store/tikv/mock-tikv/executor.go

```diff
@@ -343,28 +343,29 @@ func getRowData(columns []*tipb.ColumnInfo, colIDs map[int64]int, handle int64,
 	return values, nil
 }
 
-func extractColIDsInExpr(expr *tipb.Expr, columns []*tipb.ColumnInfo, collector map[int64]int) error {
+func extractColIDsInExpr(expr *tipb.Expr, columns []*tipb.ColumnInfo, collector []int64) ([]int64, error) {
```

Comment:
The collector should be allocated with the length of the columns beforehand, doesn't need to be returned.